artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -675,6 +675,82 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="journal-retention" maxOccurs="1" minOccurs="0">
+            <xsd:annotation>
+               <xsd:documentation>
+                  Configuration about the journal retention configuration.
+               </xsd:documentation>
+            </xsd:annotation>
+            <xsd:complexType>
+               <xsd:attribute name="unit" use="optional">
+                  <xsd:annotation>
+                     <xsd:documentation>
+                        This configures the period type to use on limit. By default it is DAYS.
+                     </xsd:documentation>
+                  </xsd:annotation>
+                  <xsd:simpleType>
+                     <xsd:restriction base="xsd:string">
+                        <xsd:enumeration value="DAYS"/>
+                        <xsd:enumeration value="HOURS"/>
+                        <xsd:enumeration value="SECONDS"/>
```

Comment:
In artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java seems we support also Minutes.